# HG changeset patch
# User Piotr Held <pjheld@gmail.com>
# Date 1431544531 21600
#      Wed May 13 13:15:31 2015 -0600
# Branch stable
# Node ID 998881b6cdefc97b594dacf75a4934d0484fb166
# Parent  ac3746f39fabb2229884d9262c291e83c1ab96ea
external.txi: Fix example case for dim_vector (bug #45100).

diff --git a/doc/interpreter/external.txi b/doc/interpreter/external.txi
--- a/doc/interpreter/external.txi
+++ b/doc/interpreter/external.txi
@@ -278,18 +278,17 @@ Matrix a;
 This can be used for all matrix and array types.
 
 @item
 Define the dimensions of the matrix or array with a dim_vector which has
 the same characteristics as the vector returned from @code{size}.  For example:
 
 @example
 @group
-dim_vector dv (2);
-dv(0) = 2; dv(1) = 3;  // 2 rows, 3 columns
+dim_vector dv (2, 3);  // 2 rows, 3 columns
 Matrix a (dv);
 @end group
 @end example
 
 This can be used on all matrix and array types.
 
 @item
 Define the number of rows and columns in the matrix.  For example:
